.github/workflows/nightly_dependency_tests.yaml

```diff
@@ -0,0 +1,76 @@
+name: Nightly dependencies@develop
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 23 * * *"
```

Comment:
It will most likely be weekly or fortnightly, and I will further recommend adding a job that needs this job and creates an issue for you. Here's how `xarray` started doing it, which is a bit complicated: https://github.com/pydata/xarray/pull/4583/ but it seems to work well: https://github.com/pydata/xarray/issues/8844

We can use a similar workflow based on this, or implement a simpler job that doesn't need to parse the logs and so on. and use existing third-party GitHub Actions that can do the same task (opening an issue, adding the workflow URL in the contents, etc.)

Reply:
since I saw the cross-link: we've since moved on to using [xarray-contrib/issue-from-pytest-log](https://github.com/xarray-contrib/issue-from-pytest-log), which abstracts away a lot of the complex code and in general is a lot more robust.

Reply:
That looks wonderful – thanks a lot, @keewis!